modules/web-console/frontend/app/modules/states/logout.state.js

```diff
@@ -27,6 +27,7 @@ angular.module('ignite-console.states.logout', [
         controller: ['Auth', function(Auth) {Auth.logout();}],
         tfMetaTags: {
             title: 'Logout'
-        }
+        },
+        template: `<div class='splash'><div class='splash-wrapper'><div class='spinner'><div class='bounce1'></div><div class='bounce2'></div><div class='bounce3'></div></div><div class='splash-wellcome'>Logout...</div></div></div>`
```

Comment:
Can be removed by appending logout().invalidateHttpSession(false) in org.apache.ignite.console.web.security.SecurityConfig#configure(org.springframework.security.config.annotation.web.builders.HttpSecurity)

Reply:
I will check

Reply:
This setting has no effect.